src/components/Profile/components/SchedulePanel/index.tsx

```diff
@@ -45,14 +49,30 @@ const GordonSchedulePanel = ({ profile, myProf }: Props) => {
       setSelectedSchedule(defaultSchedule);
 
       setLoading(false);
+
+      const getIsScheduleOpen = localStorage.getItem('isScheduleOpen');
+      if (getIsScheduleOpen !== null) {
+        setIsScheduleOpen(getIsScheduleOpen === 'true');
+      }
     });
   }, [profile.AD_Username]);
 
+  const toggleIsScheduleOpen = () => {
+    setIsScheduleOpen((wasOpen) => {
+      localStorage.setItem('isScheduleOpen', String(!isScheduleOpen));
```

Comment:
Use the constant here:
```suggestion
      localStorage.setItem(scheduleOpenKey, String(!isScheduleOpen));
```

Reply:
Thank you for the explanation, everything should be updated and simplified now, I also changed line 67 to be so the schedule starts open like suggested during the practicum.